src/components/Course/Content/ExerciseContent/index.js

```diff
@@ -114,6 +114,28 @@ const RenderContent = ({ data }) => {
     }
   }
   if (data.component === "table") {
+    // return tableData(data);
+    //Changing list data from row to column
+    const allData = data.value.map((item) => item.items);
+    const dataInRow = [];
+    for (const i in allData[0]) {
+      for (const j in allData) {
+        dataInRow.push(allData[j][i]);
+      }
+    }
+    const dataInCol = [];
+    for (const i in allData[0]) {
+      const temArray = [];
+      for (const j in dataInRow) {
+        if (j == 3) {
```

Comment:
Change is constant in software development, so you don't want to assume that the table will always have `3` columns.  Instead, you can get the number of columns in general by using `allData.length` (or `data.value.length`).